### PR TITLE
usage rights => rights & restrictions

### PR DESCRIPTION
**Pull request by jamesgorrie**
I quite likes Jonny's suggestion, so had a go and still like it.
rr-results
rr-uploads

### CODE REVIEW

File: kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="image-info">
     <div class="image-info__group">
         <dl class="image-info__wrap metadata-line image-info__usage-rights">
-            <dt class="metadata-line__key">Usage rights category</dt>
+            <dt class="metadata-line__key">Rights & restrictions</dt>
```
Comment (akash1810) [inline kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html]: `&amp;` rather than `&`?

**Comment by akash1810** [comment]
:+1:

File: kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html (hunk 1 above)
Comment (jamesgorrie) [inline kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html]: I don't  think that's necessary nowadays?

Comment (theefer) [inline kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html]: Interestingly, I just tried in the W3C validator and it's not (anymore?). `a & b` is seemingly valid. However `a &x b` is not.

**Comment by theefer** [comment]
Nice one :+1:

**Pull request merged**